public/app/features/alerting/unified/mocks/server/handlers/alertRules.ts

```diff
@@ -20,18 +21,40 @@ export const rulerRulesHandler = () => {
   });
 };
 
-export const rulerRuleNamespaceHandler = () => {
-  return http.get<{ folderUid: string }>(`/api/ruler/grafana/api/v1/rules/:folderUid`, ({ params: { folderUid } }) => {
-    // This mimic API response as closely as possible - Invalid folderUid returns 403
-    const namespace = namespaces[folderUid];
-    if (!namespace) {
-      return new HttpResponse(null, { status: 403 });
+export const rulerRuleNamespaceHandlers = () => {
+  const fetchNamespace = http.get<{ folderUid: string }>(
+    `/api/ruler/grafana/api/v1/rules/:folderUid`,
+    ({ params: { folderUid } }) => {
+      // This mimic API response as closely as possible - Invalid folderUid returns 403
+      const namespace = namespaces[folderUid];
+      if (!namespace) {
+        return new HttpResponse(null, { status: 403 });
+      }
+
+      return HttpResponse.json<RulerRulesConfigDTO>({
+        [namespaceByUid[folderUid].name]: namespaces[folderUid],
+      });
     }
+  );
 
-    return HttpResponse.json<RulerRulesConfigDTO>({
-      [namespaceByUid[folderUid].name]: namespaces[folderUid],
-    });
-  });
+  const updateNamespace = http.post<{ folderUid: string }>(
+    `/api/ruler/grafana/api/v1/rules/:folderUid`,
+    ({ params: { folderUid } }) => {
+      // This mimic API response as closely as possible.
+      // Invalid folderUid returns 403 but invalid group will return 202 with empty list of rules
```

Comment:
(Just noting - doesn't look like the 202 behaviour you're talking about is captured in this mock handler?)

Reply:
I believe this is a copy-paste mistake but we can double check when we go through the testing logic here